Engine/entry1.c

```diff
@@ -143,6 +144,7 @@ OENTRY opcodlst_1[] = {
   {  "=.T",   S(STRCPY_OP),0,     "S",    "i",
      (SUBR) strcpy_opcode_p, (SUBR) NULL, (SUBR) NULL, NULL                 },
   { "=.r",    S(ASSIGN),0,        "r",    "i",    rassign, NULL, NULL, NULL },
+  { "=.instri", S(IREF_INIT) ,0,  ":InstrDef;", "i", (SUBR) init_instr_ref},
```

Comment:
Also odd here for me as I'm not sure what this means to overload = for i to InstrDef. Prefer to have some utility opcode like find_instr_def(inum).

Reply:
There's an nstrnum overload, so I'll just remove this overload.